---
title: "Bioinformatics of Transcription Factor Binding Prediction"
authors:
- Erick I. Navarro-Delgado
- Marisol Salgado-Albarrán
- Karla Torres-Arciga
- Nicolas Alcaraz
- Ernesto Soto-Reyes
- Luis A. herrera
- Rodrigo González-Barrios
author_notes:
- "Equal contribution"
- "Equal contribition"
date: "2021-12-01T00:00:00Z"
doi: "https://doi.org/10.1201/9781003005926"

# Schedule page publish date (NOT publication's date).
publishDate: "2021-12-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["6"]

# Publication name and optional abbreviated publication name.
publication: "*Bioinformatics and Human Genomics Research*"
publication_short: ""

abstract: Transcription Factors (TFs) are crucial proteins that regulate gene expression in the cell. They bind to regulatory sequences and promote the transcription of genomic elements. The identification of TFs binding sites is often the first step towards a detailed understanding of their function, which has important implications to comprehend the cell, its development and differentiation. This chapter focuses on the bioinformatic analyses used to identify specific TF binding sites. It introduces the different high-throughput experimental techniques currently used to elucidate the regulatory elements such as, ChIP-seq, SELEX-seq, ChIP-on-chip, CUT&RUN or CUT&Tag, and the computational methods used to identify and/or predict the regulatory sequences targeted by TFs. The topics described here are the different TF binding motif representations, from Position Weight Matrices to Transcription Factor Flexible Models; the de novo motif discovery approaches, including the enumerative, probabilistic, nature-inspired, deep learning and ensemble algorithms; and motif prediction methods, which can consider phylogenetic foot printing or co-expression data, among others. An overview of the different tools used and their pros and cons in practice is discussed in order to understand the general workflow of TFBS studies and their experimental basis, both necessary to identify the limitations of the field.

# Summary. An optional shortened abstract.
summary: Book chapter detailing the bioinformatic analyses used to identify specific TF binding sites. Overview of the different tools used and their pros and cons in practice is discussed in order to understand the general workflow of TFBS studies and their experimental basis.

tags:
- Bioinformatics 
- Transcription Factor Binding Sites
featured: false

# links:
# - name: ""
#   url: ""
#url_pdf: ""
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Potential key epigenetic players in COVID-19 infection'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
#slides: example
---

<!-- 

{{% callout note %}}
# Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
# {{% /callout %}} 

{{% callout note %}}
Create your slides in Markdown - click the *Slides* button to check out the example.
{{% /callout %}}

Supplementary notes can be added here, including [code, math, and images](https://wowchemy.com/docs/writing-markdown-latex/).

-->